addons/website_project_issue/views/project_issue_templates.xml

```diff
@@ -0,0 +1,137 @@
+                    <tr t-att-class="'hidden' if len(issues) &lt; 6 else ''">
+                        <td colspan="4">
+                            <a href="" class="wp_show_more btn btn-xs btn-default"><span>Show more</span><span class="hidden">Show less</span></a>
+                        </td>
+                    </tr>
+                </table>
+            </t>
+        </xpath>
+    </template>
+
+    <template id="issues_followup" name="Project Issues">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="container">
+                    <div class="oe_structure">
+                        <div class="row">
+                            <div class="col-sm-4">
+                                <ol class="breadcrumb mt8">
+                                    <li><a href="/my/home">My Account</a></li>
+                                    <li>Issue <span t-field="issue.id"/></li>
+                                </ol>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="container">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <div class="row">
+                                    <div class="col-md-12">
+                                        <h4>Issue <span t-field="issue.id"/> ( <span t-field="issue.name"/> )</h4>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="panel-body">
+                                <div class="mb8">
+                                    <strong>Date:</strong> <span t-field="issue.create_date" t-field-options='{"widget": "date"}'/>
+                                </div>
+                                <div class='row'>
+                                    <div class="col-md-6">
+                                        <strong>Company Contact</strong>
+                                        <div t-if="issue.user_id">
+                                            <address t-field="issue.user_id" t-field-options='{"widget": "contact", "fields": ["name", "email", "phone"], "no_marker": true}'/>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-6">
+                                        <strong>Your Contact</strong>
+                                        <div t-if="issue.partner_id">
+                                            <address t-field="issue.partner_id" t-field-options='{"widget": "contact", "fields": ["name", "email"], "no_marker": true}'/>
+                                        </div>
+                                    </div>
+                                </div>
+                                <span t-field="issue.stage_id.name" class="pull-right label label-danger"/>
```

Comment:
@rim-odoo According to mock-up. I will change it if you want me to. Kindly provide your feedback.